src/chrome/content/rules/Senate.gov.xml

```diff
@@ -15,7 +15,10 @@
 
 		<!-- The rules subdomain is all 403 via TLS -->
 		<!-- As is the Senate Intelligence Committee -->
-		<exclusion pattern="^http://(?:www.)?(?:intelligence|rules)\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?intelligence\.senate\.gov/" />
+		<exclusion pattern="^http://(?:www.)?rules\.senate\.gov/" />
```

Comment:
Ah right, missed the sessions below. I agree that consistent styling is more important, then (although I suspect significant slowdowns if all the rules are rewritten this way).